core/azure-core/src/main/java/com/azure/core/implementation/serializer/jackson/JacksonAdapter.java

```diff
@@ -41,6 +41,11 @@
 
     private final XmlMapper xmlMapper;
 
+    /**
```

Comment:
```suggestion
    /*
```

Reply:
Please use the single-asterisk comment style for non-javadoc applicable locations, and use the double-asterisk comment style for javadoc applicable locations.